src/containers/app/AppFooter.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
-import { Segment, Container, List, Image } from "semantic-ui-react";
+import { Segment, List, Image } from "semantic-ui-react";
+import Container from "../../components/augmint-ui/container";
```

Comment:
any reason we are using `../../components/augmint-ui/<something>` path ? 
for me it works fine with `components/augmint-ui/<something>` that way copy/paste and folder restructure won't cause issues.
if you agree then we should change import everywhere